hummingbot/connector/exchange/ascend_ex/ascend_ex_exchange.py

```diff
@@ -1,39 +1,36 @@
-import aiohttp
 import asyncio
 import json
 import logging
 import time
-
 from collections import namedtuple
 from decimal import Decimal
 from enum import Enum
 from typing import (
+    Any,
+    AsyncIterable,
     Dict,
     List,
     Optional,
-    Any,
-    AsyncIterable,
 )
 
+import aiohttp
```

Comment:
Should be probably in the beginning of the file

Reply:
According to the sorting we agreed on, third party libraries should go in second place. In first place we should only place the Python dependencies.